src/NServiceBus.Core/Pipeline/Incoming/ReceiveFeature.cs

```diff
@@ -65,6 +66,9 @@ public Task<CompletableSynchronizedStorageSession> TryAdapt(TransportTransaction
             internal static readonly Task<CompletableSynchronizedStorageSession> EmptyResult = Task.FromResult<CompletableSynchronizedStorageSession>(new NoOpCompletableSynchronizedStorageSession());
         }
 
+        // Do not ally Fody to weave the IDisposable for us so that other threads can still access the instance of this class
```

Comment:
I'm not sure. Obviously the answer should be "no". But in this case the code is NoOp and serves no benefit other than to hide an assumption made by the core that isn't always satisfied. Any code accessing a previously disposed instance of `CompletableSynchronizedStorageSession` would already have been highlighted as a problem prior to this.

@bording typo fixed.

@timbussmann changes made.